README.md

```diff
@@ -4,13 +4,9 @@
 
 License: MIT
 
-# Usage Example
-
-    $ php example.php
-
 ## Example 1: Bordered Table (Default)
 
-    require_once 'ConsoleTable.php';
+    require "vendor/autoload.php";
```

Comment:
Can you please remove this line since it is only valid for composer users? We shall not include `require` in all examples.